numpy/lib/type_check.py

```diff
@@ -327,6 +327,11 @@ def nan_to_num(x):
     ----------
     x : array_like
         Input data.
+    copy : bool, optional
+        Whether to use a copy of `x` (True) or to replace values
+        in-place (False). The in-place operation only occurs if
+        casting to an array does not return a copy.
+        Default is True.
```

Comment:
Very small comment, but I think the standard is to add a little note saying that this is added in 1.13; look for `.. versionadded` in, e.g., `numpy/lib/stride_tricks.py` for the format.